WordPress/src/main/java/org/wordpress/android/ui/posts/prepublishing/PrepublishingPublishSettingsFragment.kt

```diff
@@ -41,8 +41,8 @@ class PrepublishingPublishSettingsFragment : PublishSettingsFragment() {
     }
 
     override fun setupContent(rootView: ViewGroup, viewModel: PublishSettingsViewModel) {
-        val closeButton = rootView.findViewById<ImageView>(R.id.close_button)
-        val backButton = rootView.findViewById<ImageView>(R.id.back_button)
+        val closeButton = rootView.findViewById<View>(R.id.close_button)
+        val backButton = rootView.findViewById<View>(R.id.back_button)
         val toolbarTitle = rootView.findViewById<TextView>(R.id.toolbar_title)
```

Comment:
I would have used the ktx approach here but then I realized because I am getting these views from the base fragment so it's not possible.